### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by xorye**
First draft for the readme.

Would it be good if I added some gifs to the readme for some of the features? 

Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,52 @@
...
+## Description
+
+This VS Code extension provides support for Quarkus development via a 
+[Quarkus language server](https://github.com/redhat-developer/quarkus-ls/tree/master/quarkus.ls)
+and a [Quarkus jdt.ls extension](https://github.com/redhat-developer/quarkus-ls/tree/master/quarkus.jdt).
+
+## Features
+
+  * Generate a Quarkus Maven project with Maven wrapper files
+  * Add Quarkus extensions to current Quarkus project
```
Comment (fbricon) [inline README.md]: to current Maven-based Quarkus project

File: README.md [hunk 2]
```diff
@@ -1,55 +1,52 @@
...
+  * Completion support for Quarkus properties in `application.properties`
+  * Hover support for Quarkus properties in `application.properties` 
+  * Retrieve Javadoc documentation for Quarkus properties in `application.properties`
```
Comment (fbricon) [inline README.md]: that's redundant with hover

**Review by fbricon — CHANGES_REQUESTED** [review]
adding a couple static images should be enough

File: README.md [hunk 3]
```diff
@@ -1,55 +1,52 @@
...
+
+This VS Code extension provides support for Quarkus development via a 
+[Quarkus language server](https://github.com/redhat-developer/quarkus-ls/tree/master/quarkus.ls)
+and a [Quarkus jdt.ls extension](https://github.com/redhat-developer/quarkus-ls/tree/master/quarkus.jdt).
+
+![](images/applicationProperties.png)
+
+## Features
+
+  * Generate a Quarkus Maven project with Maven wrapper files
```
Comment (angelozerr) [inline README.md:12]: I think we should separate features for generate Quarks project and for application.properties. We could have 

## Quarkus project support

## application.properties support

Comment (xorye) [reply README.md:12]: How's this?

image

**Comment by fbricon** [comment]
Very nice! Thanks @xorye

**Pull request merged**